Script/run.js

```diff
@@ -82,7 +82,7 @@ class AppState {
   }
 
   setWindowActive() {
-    document.cookie = `actw=${window.name}`;
+    document.cookie = "actw=${window.name}; SameSite:Lax";
```

Comment:
```suggestion
    document.cookie = `actw=${window.name}; SameSite:Lax`;
```

The ${window.name} template var won't work unless backtick string is used